editoast/fga_migrations/src/migration.rs

```diff
@@ -0,0 +1,509 @@
+use std::sync::LazyLock;
+
+use crate::client::CliParser;
+use crate::client::Migration as TargetMigration;
+use fga::client::Client;
+use fga::client::ConnectionSettings;
+use fga::client::InitializationError;
+use fga::client::QueryError;
+use fga::client::RequestFailure;
+use fga::client::TooManyTuples;
+use fga::compile_model;
+use fga::model::Relation;
+use indexmap::IndexMap;
+use indexmap::indexmap;
+use itertools::Either;
+use sha1::Digest;
+use sha1::Sha1;
+use tracing::error;
+use tracing::info;
+use tracing::instrument;
+use url::Url;
+
+const MIGRATIONS_MODEL: &str = include_str!("../migrations_model.fga");
+
+#[cfg(not(test))]
+static OPENFGA_MIGRATIONS: LazyLock<IndexMap<&'static str, &'static str>> = LazyLock::new(|| {
+    // The migrations declaration order matches the order in which they will be applied
+    indexmap! {
```

Comment:
```suggestion
    #[cfg(not(test))]
	let map = indexmap! {};
	#[cfg((test))]
	let map = indexmap! {};
	map
```

You might not need to repeat the whole signature.